crates/interfaces/src/db.rs

```diff
@@ -6,46 +6,39 @@ use thiserror::Error;
 pub enum DatabaseError {
     /// Failed to open the database.
     #[error("failed to open the database: {0}")]
-    Open(Box<DatabaseErrorInfo>),
+    Open(DatabaseErrorInfo),
     /// Failed to create a table in the database.
     #[error("failed to create a table: {0}")]
-    CreateTable(Box<DatabaseErrorInfo>),
+    CreateTable(DatabaseErrorInfo),
     /// Failed to write a value into a table.
     #[error(transparent)]
-    Write(#[from] Box<DatabaseWriteError>),
+    Write(#[from] DatabaseWriteError),
```

Comment:
can we keep this boxed? We don't want to impact the size of RethError and this variant is the biggest.

Reply:
Fixed.